easybuild/easyconfigs/n/neovim/neovim-0.11.4-gcccore-13.2.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'neovim'
+version = '0.11.4'
+
+homepage = 'http://neovim.io'
+description = """Vim-fork focused on extensibility and usability"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+sources = [{
+    'filename': 'neovim-0.11.4.tar.gz',
+    'git_config': {
+            "url": "https://github.com/neovim",
+            "repo_name": "neovim",
+            "commit": "cec0ecabd8f47ff81dcb52e8fc9003e365563a84",
+            'recursive': True,
+            'keep_git_dir': True,
+    },
+}]
+
+builddependencies = [('CMake', '3.27.6')]
```

`binutils` should be a build-time dependency, when compiling software with the `GCCcore` toolchain
```suggestion
builddependencies = [
    ('binutils', '2.40'),
    ('CMake', '3.27.6'),
]
```